formula-one-tracker.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>

```

Comment:
One pro-tip: If you named this `index.html`, it would load at the root of the server (http://127.0.0.1:8080)